www/_posts/2024-05-23-cordova-android-13.0.0.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 13.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 13.0.0`! This is one of Cordova's supported platforms for building Android applications.
+
+* [cordova-android@13.0.0](https://www.npmjs.com/package/cordova-android)
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@13.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add android@13.0.0
+```
+
+## Release Highlights
+
+### BREAKING CHANGES
+
+* **Increased Target SDK**
+
+    This release has increased the target SDK to 34 (Android 14).
+
+* **Increased Android Studio Requirement**
+
+    Android Studio Jellyfish is required if building and running from Android Studio.
+
+* **Required Build Tools**
+
+    To use `cordova-android@13`, SDK Platform `34` and SDK Build Tools `34.0.0` must be installed. Older build tools version can be uninstalled if older versions of cordova-android is no longer used in your projects.
+
+    To install SDK Platform 34:
+
+    1. Open Android Studio's **SDK Manager**:
+    2. Click on `SDK Platforms` tab
+    3. Check `Android 14.0 ("UpsideDownCake")` which has the `API Level` of `34`
+    4. Click `Apply`
+
+    <img src="{{ site.baseurl }}/static/img/blog/2024/cordova-android-13-platform-sdk-34.png" style="width: 100%;" alt="Android SDK Platform" />
+
+    To install SDK Build Tools 34.0.0:
+
+    5. Open Android Studio's **SDK Manager**:
+    6. Click on `SDK Tools` tab
+    7. Check `Show Package Details`
+    8. Expand `Android SDK Build-Tools`
+    9. Check `34.0.0`
+    10. Click `Apply`
+
+    <img src="{{ site.baseurl }}/static/img/blog/2024/cordova-android-13-sdk-build-tools-3400.png" style="width: 100%;" alt="Android SDK Build Tools" />
+
+* **Project Dependencies**
+
+    The following project dependencies were bumpped:
+
+    * Kotlin: `1.9.24`
+    * Gradle: `8.7`
+    * Android Gradle Plugin (AGP): `8.3.0`
+
+    If you or a plugin has made changes to any of the following configuration preferences, the build results might not match the expected outcomes with this release:
+
+    * `android-minSdkVersion`
+    * `android-maxSdkVersion`
+    * `android-targetSdkVersion`
+    * `android-compileSdkVersion`
+    * `android-buildToolsVersion`
+    * `GradleVersion`
+    * `AndroidGradlePluginVersion`
+    * `GradlePluginKotlinVersion`
```

Comment:
Is it worth mentioning what the default values are, or linking to where they are all defined?

We're basically telling people "If you edited any of these, please review the new settings" but not giving any pointer to where to find those new settings.

Reply:
Perhaps we can give a table:

|Preference|Default Value|
|---|---|
|`android-minSdkVersion`|24|
|`android-maxSdkVersion`|_Not set_|
|`android-targetSdkVersion`|34|
|`android-compileSdkVersion`|_android-targetSdkVersion configured value_|
|`android-buildToolsVersion`|"34.0.0"|
|`GradleVersion`|"8.7"|
|`AndroidGradlePluginVersion`|"8.3.0"|
|`GradlePluginKotlinVersion`|"1.9.24"|

Markdown:

```markdown
|Preference|Default Value|
|---|---|
|`android-minSdkVersion`|24|
|`android-maxSdkVersion`|_Not set_|
|`android-targetSdkVersion`|34|
|`android-compileSdkVersion`|_android-targetSdkVersion configured value_|
|`android-buildToolsVersion`|"34.0.0"|
|`GradleVersion`|"8.7"|
|`AndroidGradlePluginVersion`|"8.3.0"|
|`GradlePluginKotlinVersion`|"1.9.24"|
```